htdocs/variants/combinations.php

```diff
@@ -137,9 +137,10 @@
 		// for conf PRODUIT_MULTIPRICES
 		if ($conf->global->PRODUIT_MULTIPRICES) {
 			$level_price_impact = array_map('price2num', $level_price_impact);
+			$level_price_impact_percent = array(1 => $price_impact_percent);
```

Comment:
Why only the key 1 is filled? Later we used the array with indic [$i] where is the price level. I suggest
```
$level_price_impact_percent = array();
for ($i = 1; $i <= $conf->global->PRODUIT_MULTIPRICES_LIMIT; $i++) {
$level_price_impact_percent = array($i, $price_impact_percent);
} 
```

Reply:
I’ll check this with John next time I see him, because I am not very familiar with multiprice.

Reply:
I (finally) discussed it with John: seems OK at first look, but we want to review it once more to be sure.